superset-frontend/spec/javascripts/dashboard/components/nativeFilters/FilterBar_spec.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import { supersetTheme, ThemeProvider } from '@superset-ui/core';
+import FilterBar from 'src/dashboard/components/nativeFilters/FilterBar';
+import { mockStore } from 'spec/fixtures/mockStore';
+
+describe('FilterBar', () => {
+  const props = {
+    filtersOpen: false,
+    toggleFiltersBar: jest.fn(),
+  };
+
+  const wrapper = mount(
+    <Provider store={mockStore}>
+      <FilterBar {...props} />
+    </Provider>,
+    {
+      wrappingComponent: ThemeProvider,
+      wrappingComponentProps: {
+        theme: supersetTheme,
+      },
+    },
```

Comment:
```suggestion
    </Provider>,
```

Reply:
This (with the import changes above) is basically what I was suggesting on the other test.